_stories/getting-started/Customization/README.md

```diff
@@ -0,0 +1,46 @@
+## Customization
+(using the Design System npm package)
```

Comment:
```suggestion
When using GumDrops with the GumGum Design System npm package, you can customize the colors and styles to use any colors and styles you want, instead of or in addition to the default themes.
```